docker/edge-alpine.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine as base
+RUN apk add --no-cache nodejs yarn npm python3 openssl build-base
+WORKDIR /app
+ADD .yarn ./.yarn
+ADD yarn.lock package.json .yarnrc.yml ./
+RUN yarn workspaces focus --all --production
+
+# Since we’re just using static files, use the Ubuntu image to build the frontend
+# (otherwise electron fails to build)
+FROM node:16-bullseye as frontend
+WORKDIR /frontend
+# Rebuild whenever there are new commits to the frontend
+ADD "https://api.github.com/repos/actualbudget/actual/commits" /tmp/actual-commit.json
+RUN git clone --depth=1 https://github.com/actualbudget/actual /frontend
```

Comment:
Hmm...could be docker magic I don't know...but is this URL correct? If so, how?? 😸

Reply:
What do you mean?

Reply:
Oh shoot -- it wrapped just right in side-by-side mode so that I thought I was seeing `https://github.com/actualbudget/actual/frontend` rather than two arguments. 
Ignore me!